src/NumericalAlgorithms/DiscontinuousGalerkin/BoundarySchemes/FirstOrder/BoundaryData.hpp

```diff
@@ -20,9 +20,8 @@ namespace FirstOrderScheme {
 namespace detail {
 template <typename NumericalFluxType>
 struct BoundaryDataImpl {
-  static_assert(tt::conforms_to_v<NumericalFluxType, protocols::NumericalFlux>,
-                "The 'NumericalFluxType' must conform to the "
-                "'dg::protocols::NumericalFlux'.");
+  static_assert(
```

Comment:
Not a change request, but a point of discussion.

Here and in several other places for the numerical fluxes you check explicitly that they `assert_corform_to`. In contrast, the protocols documentation says
```
We are currently testing protocol conformance as part of our unit tests, so that
the `tt::conforms_to` metafunction only needs to check if a type inherits off
the protocol, but doesn't need to check the protocol's implementation. This is
both to keep compile times low and to have a SFINAE-friendly way of checking
protocol conformance. Full protocol conformance is tested with the
`tt::assert_conforms_to` metafunction.
```
which I interpreted as saying "You should only use `assert_conforms_to` in tests. How do you justify/remedy this apparent inconsistency?

Reply:
I re-formulated that part of the documentation to this:

> The `tt::conforms_to` metafunction only checks if the class _indicates_ it
conforms to the protocol. Where SFINAE-friendliness is not necessary prefer the
`tt::assert_conforms_to` metafunction that triggers static asserts with
diagnostic messages to understand why the class does not conform to the
protocol.

Is that more clear?